web/ui/templates/targets.html

```diff
@@ -44,10 +44,10 @@ <h2 id="targets">Targets</h2>
                         </span>
                       </td>
                       <td>
-                        <span class="cursor-pointer" data-toggle="tooltip" title="" data-html=true data-original-title="<b>Before relabeling:</b>{{range $k, $v := .DiscoveredLabels}}<br>{{$k | html | html}}=&quot;{{$v | html | html}}&quot;{{end}}">
+                        <span class="cursor-pointer" data-toggle="tooltip" title="" data-html=true data-original-title="<b>Before relabeling:</b>{{range $k, $v := .DiscoveredLabels}}<br>{{$k}}=&quot;{{$v}}&quot;{{end}}">
```

Comment:
It was used twice because when you put an HTML-escaped value into the `data-original-title` attribute, it apparently already gets HTML-unescaped automatically by the browser before being put into the tooltip overlay. I tried adding a single `| html` there again, which doesn't cause the template execution to fail, but still doesn't fix the XSS issue.

I'd have to dig deeper myself to see what the proper solution is here.